src/main/java/net/dries007/tfc/common/blocks/TFCBlocks.java

```diff
@@ -241,6 +238,19 @@ public final class TFCBlocks
         )
     );
 
+    public static final Map<Wood, Map<Metal.Default, Map<Class<? extends ITFCHangingSignBlock>, RegistryObject<SignBlock>>>> HANGING_SIGNS = Helpers.mapOfKeys(Wood.class, wood -> {
```

Comment:
Is there a reason we are cludging these into a single map? None of our block registrations look like this and it just seems to have a needlessly large code footprint